datahub-web-react/src/app/analytics/event.ts

```diff
@@ -120,6 +121,19 @@ export enum EventType {
     HomePageClick,
     SearchBarFilter,
     ClickProductUpdate,
+    HomePageTemplateModuleCreate,
```

Comment:
we could add one more event - `HomePageTemplateModuleCancel` for when they start creating a module but then cancel and don't save anything